src/main/resources/mybatis/lookup.xml

```diff
@@ -78,7 +78,7 @@
         select featureid
           from nhdplus.catchmentsp
             where st_intersects(nhdplus.catchmentsp.the_geom,
-              st_setsrid(st_makepoint(#{lon}, #{lat}), 4269)::geography);
+              st_geomfromtext('POINT(${lon} ${lat})', 4269));
```

Comment:
While I am making suggestions outside this pull request, how about a couple of sentences in the team wiki
on how to write these type of queries? Hopefully, avoid this issue in the future with other developers.
Thanks.